auth0/src/main/res/values/strings.xml

```diff
@@ -25,7 +25,4 @@
 <resources>
     <string name="com_auth0_webauth_permission_missing_title">Missing permissions</string>
     <string name="com_auth0_webauth_permission_missing_description">Some permissions required by this provider were not granted. You can try to authenticate again or go to the application\'s permission screen in the phone settings and grant them. The missing permissions are:\n%s</string>
-
-    <string name="com_auth0_webauth_network_error">Network error</string>
-    <string name="com_auth0_webauth_webview_loading_retry">Retry</string>
```

Comment:
These values were only used by the Web view activity, inspection identified these as now unused.